src/main/java/TaskList.java

```diff
@@ -0,0 +1,43 @@
+public class TaskList {
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+
+    public int getTaskCount(){
+        return taskCount;
+    }
+
+
+    public void addTask(Task task){
+        if(taskCount == 100){
+            System.out.println("Duke can store 100 task at most.");
+        } else{
+            tasks[taskCount] = task;
+            taskCount++;
+        }
+    }
+
+    public void setTaskDone(int taskIndex){
+        tasks[taskIndex].setTaskDone(true);
```

Comment:
perhaps you can rename this setTaskDone to something else? I almost thought it looked like a recursive function until I noticed the method setTaskDone in the Task class also